### DISCUSSION TITLE
[v2]About Platform-specific Configuration

### DISCUSSION THREAD

**Discussion by nongyehong** [tauri-apps/tauri#11079] [Q&A]
![QQ_1726884759921](https://github.com/user-attachments/assets/4181c937-1e55-487f-8b05-e6f23442d1be)
Let's say I currently modify the configuration in the tauri.macos.conf.json on the mac system, for example, I changed the height of the configuration of the creation window, but I didn't change it after restarting the project, and when I changed tauri.conf.json unrelated content, I restarted it again, and the previous changes in the tauri.macos.conf.json took effect

**Comment by FabianLars** [comment]
https://github.com/tauri-apps/tauri/issues/10963
